decidim-core/app/helpers/decidim/scopes_helper.rb

```diff
@@ -14,8 +14,8 @@ module ParticipatoryProcessScopesHelper
     # participatory_process - a Decidim::ParticipatoryProcess.
     #
     # Returns an Array.
-    def participatory_process_subscopes(participatory_process)
-      [Option.new("", I18n.t("decidim.participatory_processes.scopes.global"))] +
+    def subscopes_for(participatory_process)
```

Comment:
Maybe you can rename participatory_process with scopable and update the upper comment with Decidim::Scopable instead of Decidim::ParticipatoryProcess?